content/community/community.md

```diff
@@ -3,11 +3,47 @@ title: Community
 description: The TiKV community
 ---
 
-TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to [contribute here](/docs/3.0/reference/contribute/).
+TiKV is an open source project with an active community. TiKV's source code is hosted on [GitHub](https://github.com/tikv/tikv) and is a [Cloud Native Computing Foundation](https://www.cncf.io) incubating project. We have information about how to contribute [here](/community/contribute/).
 
-The TiKV community is organised into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is below, follow the links for more information.
+## SIGs
+
+The TiKV community is organized into special interest groups (SIGs). More information about the community and its governance is in the [community repository](https://github.com/tikv/community). The current roster of SIGs is listed below. follow the links for more information.
 
 * [Coprocessor](/community/sig-coprocessor/)
 * [Engine](/community/sig-engine/)
 * [Raft](/community/sig-raft/)
 * [Transaction](/community/sig-transaction/)
+
+## Communications
+
+In TiKV community we value open communication and open collaboration. Here are the general communication channels we use to communicate.
+
+- **General contact methods**
+
+    - [Slack](https://join.slack.com/t/tikv-wg/shared_invite/enQtNTUyODE4ODU2MzI0LWVlMWMzMDkyNWE5ZjY1ODAzMWUwZGVhNGNhYTc3MzJhYWE0Y2FjYjliYzY1OWJlYTc4OWVjZWM1NDkwN2QxNDE) (Recommended)
+    - [Twitter](https://twitter.com/tikvproject)
+    - [Reddit](https://www.reddit.com/r/TiKV)
```

Comment:
We should not promote Reddit - the forum is basically unused and Reddit has a poor reputation